src/components/approval-type/ApprovalType.vue

```diff
@@ -141,64 +147,45 @@ export default class ApprovalType extends Vue {
   public validate (): boolean {
     if (this.approvalTypeSelected === this.ApprovalTypes.VIA_COURT_ORDER) {
       let status = this.$refs.courtNumRef.validate()
-      this.$emit('emitValid', status)
+      this.$emit('valid', status)
       return status
     } else if (this.approvalTypeSelected === this.ApprovalTypes.VIA_REGISTRAR) {
       // Emit true (valid) if both dates were selected. Emit false (invalid) if at least one was empty.
       let status = (!!this.noticeDateText && !!this.applicationDateText)
-      this.$emit('emitValid', status)
+      this.$emit('valid', status)
       return status
     } else {
       // Default state. Emit false (invalid).
       return false
     }
   }
 
-  /** Validations rules for the date picker fields. */
-  private setDateRules (): void {
-    this.datePickerRules = [
-      (v: string) => !!v || 'Date is required'
-    ]
-  }
-
   private emitValidationError (event: boolean): void {
-    this.$emit('emitValid', !event)
+    this.$emit('valid', !event)
   }
 
   // Emit the approval type (radio button selected).
-  @Emit('emitRadioButtonChange')
+  @Emit('radioButtonChange')
   private radioButtonChanged (event: string): void {
     if (event === this.ApprovalTypes.VIA_REGISTRAR) {
-      Vue.set(this, 'courtOrderNumber', '')
+      Vue.set(this, 'courtOrderNumberText', '')
       Vue.set(this, 'courtOrderNumRules', [])
```

Comment:
I think neither of these Vue.set() is needed because reactivity will work fine if you just reassign your local variables to the new values.

As I understand it, Vue.set() is used when the an item/property is added to an array/object. This is because the local variable still points to the same object, so reactivity needs to be triggered.

Another way to handle to handle a new item/property to an existing array/object this is to reassign the local variable to a NEW object.

Anyway, please check if this code works with a simple assignment instead of Vue.set, and if so then simply it.